contrib/mesos/pkg/executor/executor.go

```diff
@@ -88,24 +86,13 @@ func (s *stateType) transitionTo(to stateType, unless ...stateType) bool {
 	}
 }
 
-type kuberTask struct {
-	mesosTaskInfo *mesos.TaskInfo
-	launchTimer   *time.Timer // launchTimer expires when the launch-task process duration exceeds launchGracePeriod
-	podName       string      // empty until pod is sent to kubelet and registed in KubernetesExecutor.pods
-}
-
-type podStatusFunc func() (*api.PodStatus, error)
-
 // KubernetesExecutor is an mesos executor that runs pods
 // in a minion machine.
 type Executor struct {
-	updateChan           chan<- kubetypes.PodUpdate // sent to the kubelet, closed on shutdown
 	state                stateType
-	tasks                map[string]*kuberTask
-	pods                 map[string]*api.Pod
 	lock                 sync.Mutex
```

Comment:
yes it's used to coordinate and protect other things, so can't get rid if it just yet